framework/src/org/checkerframework/framework/util/element/SuperTypeApplier.java

```diff
@@ -81,7 +81,16 @@ public int getElementIndex() {
      */
     @Override
     public int getTypeCompoundIndex(Attribute.TypeCompound anno) {
-        return anno.getPosition().type_index;
+        int type_index = anno.getPosition().type_index;
+        // TODO: this is a temporary workaround of a bug in jsr308-langtools
```

Comment:
Yes, it's just a langtools bug.  Here's the OpenJDK bug report: https://bugs.openjdk.java.net/browse/JDK-8164519.  Could you please change `jsr308-langtools` to just `langtools` on this line and add the link to the bug report?